tests/lib/snaps/store/test-snapd-content-plug-to-classic-slot/bin/content-plug

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -e
+
+cat <<EOF
+Please run:
+sudo snap connect content-plug:shared-content-plug content-slot:shared-content-slot
+if you see an permission denied error
```

Comment:
would the client without this connection really see a `permission denied` error? I would assume they would see something like `No such file or directory` since the file just wouldn't exist in the snap's mount namespace

Reply:
also given that this is a test snap used for this test and this test only, do we expect folks to need to see a warning like this? I think it's unlikely anyone will use this test for anything else, so you could probably just remove this warning

Reply:
Removed now.